codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -91,11 +93,25 @@ impl Schema {
         Self::new(entities, enums)
     }
 
-    pub fn parse_from_file(path_to_schema: &PathBuf) -> anyhow::Result<Self> {
-        let schema_string = std::fs::read_to_string(&path_to_schema).context(format!(
+    pub fn parse_from_file(
+        project_paths: &ParsedProjectPaths,
+        maybe_custom_path: &Option<String>,
```

Comment:
I wonder if this shouldn't somehow be part of ParsedProjectPaths 🤔

Reply:
Hmmm, that sounds rational. If I ever come to the part of the codebase, I'll think of the possibility of changing it